src/base.ts

```diff
@@ -74,12 +76,14 @@ export interface ApiCallOptions extends Omit<RequestInit, "method" | "body"> {
 
 interface AuthorizedApiCallOptions extends ApiCallOptions {
   isAuthorizedRequest?: boolean
+  apiVersion?: ApiVersion
```

Comment:
This has nothing to with authorized API call, can you add it on base `ApiCallOptions` instead?